package.json

```diff
@@ -15,32 +15,19 @@
     "test": "npm run lint && tsc && cp -r src/stylesheets lib && mocha --require lib/testHelper.js lib/__tests__/*.js lib/**/__tests__/*.js lib/**/**/__tests__/*.js",
     "test-file": "npm run lint && tsc && cp -r src/stylesheets lib && mocha --require lib/testHelper.js",
     "test-browser": "npm run test-chrome && npm run test-firefox",
```

Comment:
This one should also be removed.

Reply:
Which one--`test-browser`?

Reply:
Yes - it calls the ones you deleted already.